cli/asconfig.js

```diff
@@ -0,0 +1,178 @@
+const path = require("path");
```

Comment:
Suggesting to make this file `cli/util/config.js` with a corresponding `.d.ts`.

Reply:
I will implement the d.ts file last step because I haven't decided on an API yet.

Reply:
I also don't care where the code ends up. This is just a proof of concept at this point, and I want to make sure we are on the same page first.